src/components/userPage/components/constants.js

```diff
@@ -0,0 +1,124 @@
+export const NAVBAR = {
+    LINKS: {
+        HOME: '/',
+        SUPPORT: "/about/",
+    },
+    STINGS: {
+        CATEGORIES: "Categories",
+        FAQS: "FAQs",
+        HI_USER_ICON: "hi-user-icon",
+        HI_USER_TEXT: "Hi user!",
+        HOME_TEXT: "Home",
+        SUPPORT_TEXT: "Support",
+
+    },
+};
+
+//app logo
+export const LOGO = {
+    SOURCE:  "/images/Agroeasylogo.png",
+    WIDTH: "60px",
+};
+
+/** 
+//Producer profile
+export const FAKE_USER = {
+    address: '24, Wilkie Crescent, Ikeja',
+    city: 'Lagos',
+    country: 'Nigeria',
+    email: 'joshuaobinna93@gmail.com',
+    firstName: 'Josh',
+    gender: 'male',
+    lastName: 'Obinna',
+    phoneNumber: '+2347061589692',
+};
+*/
+
+//userpage string
+export const USER_PAGE = {
+    CLASSNAMES: {
+        AVATAR: 'avatar',
+        AVATAR_CARD: 'avatar-card',
+        BASIC_INFO: 'basic-info',
+        BIG_CARD: 'big-card',
+        BIG_CONTENT: 'big-content',
+        BIG_LAYOUT: 'big-layout',
+        CARD_HEADER: 'card-header',
+        CARD_META: 'card-meta',
+        DATA_TITLE: 'data-title',
+        HEADER_TITLE: 'header-title',
+        INFO_CARD: 'info-card',
+        INNER_CARD: 'inner-card',
+        ITEM_LAYOUT: 'horizontal',
+        MENU: 'menu',
+        PROFILE_CONTENT: 'profile-content',
+        SIDER: 'sider',
+        SM_CONTENT: 'sm-content',
+    },
+    PROFILE_PIX: "./images/josh.jpg",
+    STRINGS: {
+        CENTER: 'center',
+        DARK: 'dark',
+        FLEX: 'flex',
+        FOLDER: 'folder',
+        INFO_CIRCLE: 'info-circle',
+        INLINE: 'inline',
+        MESSAGE: 'message',
+        NOTIFICATION: 'notification',
+        ORIENTATION: 'left',
+        SETTING: 'setting',
+        SHOPPING: 'shopping',
+        SM_HEADER: 'sm-header',
+        SM_HEADER_DIV: 'sm-header-div',
+        SOLUTION: 'solution',
+        SQUARE: 'square',
+        START: 'start',
+    },
+    TEXTS: {
+        ACCOUNT_INFO: 'Account Info',
+        BASIC_INFO_TEXT: 'BASIC INFORMATION',
+        CONTACT_INFO_TEXT: 'CONTACT INFORMATION',
+        HELP: 'Help',
+        LOCATION_INFO_TEXT: 'LOCATION',
+        MESSAGES: 'Messages',
+        NOTIFICATIONS: 'Notifications',
+        PURCHASES: 'Purchases',
+        SAVED_ITEMS: 'Saved Items',
+        SETTINGS: 'Settings',
+        USER_PROFILE: 'User Profile',
+    },
+};
+
+export const DATA = [
```

Comment:
call this `BASIC_INFO`

Reply:
there's gonna be name conflict. lemme call it BASIC_INFOMATION